src/components/BottomBar.jsx

```diff
@@ -4,32 +4,85 @@ import {
   EuiFlexGroup,
   EuiFlexItem,
   EuiText,
+  EuiLink,
 } from "@elastic/eui";
 import React from "react";
 
 export default class Bottombar extends React.Component {
   render() {
+    const socialIcons = {
+      fontSize: "30px",
+    };
     return (
       <>
         <EuiBottomBar position="static">
-          <EuiFlexGroup direction="row" alignItems="center">
-            <EuiFlexItem>
-              <EuiText>
-                <h4>Hosted by Elastic's Rainbow Stack ERG </h4>
-              </EuiText>
+          <EuiFlexGroup alignItems="center" justifyContent="spaceBetween">
+            <EuiFlexItem grow={false}>
+              <EuiText>Hosted by Elastic's Rainbow Stack ERG</EuiText>
               <EuiText>Group for LGBTQIA+ members and allies</EuiText>
             </EuiFlexItem>
-
             <EuiFlexItem grow={false}>
-              <EuiButton color="success" fill href="#" target={"_blank"}>
-                Sign up for updates
-              </EuiButton>
+              <EuiFlexGroup
+                direction="row"
+                alignItems="center"
+                justifyContent="center"
+                responsive={false}
+              >
+                <EuiFlexItem grow={false}>
+                  <EuiLink href="#">
```

Comment:
Quick change - but can you add the `external` prop to this or `target="_blank"` to each of these `EuiLinks` so that these open in a new tab when clicked?